### ISSUE TITLE
[Feature Request] Add an interest calculation feature for the saving account.

### ISSUE THREAD

**Issue by f97** [mayswind/ezbookkeeping#30]
Add a feature to display the start date, term, and interest amount, and automatically notify when the interest payment date arrives.

Sample example: 

![IMG_0221](https://github.com/user-attachments/assets/dc4a06a2-6e49-4676-a650-be32a11e56f8)

**Comment by mayswind** [comment]
I'm afraid it would be very difficult for a third-party app to provide the same calculation results as every bank's actual interest amount. I think if it's difficult to provide accurate results, it's better not to provide this feature.

**Comment by f97** [comment]
> I'm afraid it would be very difficult for a third-party app to provide the same calculation results as every bank's actual interest amount. I think if it's difficult to provide accurate results, it's better not to provide this feature.

No, I think we should just add it to our application. When entering data, we will include fields for interest rate and interest payment date... and when the payment date arrives, we will notify the user. (telegram bot is an option).

**Comment by mayswind** [comment]
> > I'm afraid it would be very difficult for a third-party app to provide the same calculation results as every bank's actual interest amount. I think if it's difficult to provide accurate results, it's better not to provide this feature.
> 
> No, I think we should just add it to our application. When entering data, we will include fields for interest rate and interest payment date... and when the payment date arrives, we will notify the user. (telegram bot is an option).

Do you mean to just add more information to the saving account (including reminders), but not let ezbookkeping automatically generate a new transaction?

**Comment by f97** [comment]
> Do you mean to just add more information to the saving account (including reminders), but not let ezbookkeping automatically generate a new transaction?



> > > I'm afraid it would be very difficult for a third-party app to provide the same calculation results as every bank's actual interest amount. I think if it's difficult to provide accurate results, it's better not to provide this feature.
> > 
> > 
> > No, I think we should just add it to our application. When entering data, we will include fields for interest rate and interest payment date... and when the payment date arrives, we will notify the user. (telegram bot is an option).
> 
> Do you mean to just add more information to the saving account (including reminders), but not let ezbookkeping automatically generate a new transaction?

If ezbookkeping can automatically generate a new transaction and add a notification, it would be better to let usrs know it has been created.

**Comment by mayswind** [comment]
> If ezbookkeping can automatically generate a new transaction and add a notification, it would be better to let usrs know it has been created.

I think it is difficult for ezbookkeeping to calculate interest rates exactly the same as the bank's, but storing some account information and send reminders can be a new feature.
